client-app/src/common/models/search/keyword-search-query.ts

```diff
@@ -0,0 +1,20 @@
+import { KeywordSearchCriteria } from './keyword-search-criteria';
+import { SearchQuery } from './search-query';
+
+/* eslint @typescript-eslint/camelcase: ["error", {properties: "never"}] */
+export class KeywordSearchQuery extends SearchQuery {
+  constructor(
+    public page?: string,
+    public page_size?: string,
```

Comment:
Because search query is an object to which query builder deserialize query. So I just use here names which supported by storefront. `q` is the same as keyword.
https://github.com/VirtoCommerce/vc-storefront-core/blob/master/VirtoCommerce.Storefront.Model/Catalog/ProductSearchCriteria.cs#L90-L96